ngraph/core/include/ngraph/descriptor/tensor.hpp

```diff
@@ -63,6 +67,8 @@ namespace ngraph
             HostTensorPtr get_lower_value() const { return m_lower_value; }
             /// \brief gets upper bound value description
             HostTensorPtr get_upper_value() const { return m_upper_value; }
+            /// \brief gets upper bound value description
+            std::vector<std::string> get_value_label() const { return m_value_label; }
```

Comment:
If you introduced `using TensorLabel`, it will be better to use it for all usages. Or we should remove this `using`